packages/cli/README.md

```diff
@@ -1,32 +1,31 @@
+This is an experimental implementation an Ultralight node packaged as a NodeJS application and run from the command line.  
 
 ## Usage
 
-### Quickstart
-- Clone the ultralight monorepo and run `npm i` from the monorepo root.
-- Change to `./packages/cli`
-- Run `node dist/index.js --proxy --nat=extip --bootnode=enr:[Your favorite bootnode ENR here (if desired)] `
-- The node will start a local instance of the network proxy, determine its public IP address, and then connect to the bootnode specified
+The easiest way to get started with Ultralight is to use the `npm run dev` script to start up a a single local node:
+
+`npm run dev` along with any of the below parameters.
+
 ### CLI Parameters
- - `--pk`              base64 string encoded protobuf serialized private key 
- - `--bootnode`        base64 string encoded ENR of Bootnode  (e.g. `enr:-IS...`)
- - `--bootnodeList`    path to a file containing a list of bootnode ENRs (note: one ENR per line)
- - `--proxy`           Start proxy service  [default: true]          
- - `--nat`             NAT Traversal options for proxy [choices: "localhost", "lan", "extip"] 
--  `--persistentPort`  run the proxy on a persistent UDP port             
--  `--rpc`             Enable the JSON-RPC server with HTTP endpoint [default: true]
--  `--rpcPort`         HTTP-RPC server listening port             
--  `--rpcAddr`         HTTP-RPC server listening interface address [default: "localhost"]
--  `--metrics`         Turn on Prometheus metrics reporting [default: false]
--  `--metricsPort`     Port exposed for metrics scraping  [default: 18545]
--  `--dataDir`         data directory where content is stored        
-### Starting with the same Node ID 
+  --pk                base64 string encoded protobuf serialized private key  [string]
+  --bootnode          ENR of Bootnode                                        [string]
+  --bindAddress       initial IP address and UDP port to bind to             [string]
+  --bootnodeList      path to a file containing a list of bootnode ENRs      [string]
+  --rpc               Enable the JSON-RPC server with HTTP endpoint          [boolean] [default: true]
+  --rpcPort           HTTP-RPC server listening port                         [number] [default: 8545]
+  --rpcAddr           HTTP-RPC server listening interface address            [string]
+  --metrics           Turn on Prometheus metrics reporting                   [boolean] [default: false]
+  --metricsPort       Port exposed for metrics scraping                      [number] [default: 18545]
+  --dataDir           data directory where content is stored                 [string]
+  --web3              web3 JSON RPC HTTP endpoint for local Ethereum node    [string]
+  --networks          subnetworks to enable  (options are: `history`, `state`, `beacon`) [default: `history`]
```

Comment:
Bonus suggestion, also 1:1 counting for our client: I think it would be much cooler to have "direct" networks options - since so prominet and important - so `--history`, `--state`, `--beacon`. 🤩 😋

Again, same for EthereumJS client (`--holesky`,...).